tests/model_monitoring/test_helpers.py

```diff
@@ -607,3 +608,55 @@ def test_get_output_stream_unsupported() -> None:
                 name="k-tgt", brokers="localhost", topic="t1"
             ),
         )
+
+
+def test_get_start_end():
+    now = mlrun.utils.datetime_now()
+
+    # Test default when only end is provided
+    start, end = get_start_end(
+        start=None,
+        end=now,
+    )
+
+    assert start == mlrun.utils.datetime_min()
+    assert end == now
+
+    # Test when delta is provided
+    start, end = get_start_end(
+        start=None,
+        end=now,
+        delta=datetime.timedelta(seconds=1),
+    )
+
+    assert start == now - datetime.timedelta(seconds=1)
+
+    # Test when start, end and delta are provided (in this case delta should be ignored)
+    start, end = get_start_end(
+        start=now - datetime.timedelta(seconds=10),
+        end=now,
+        delta=datetime.timedelta(seconds=1),
+    )
+
+    assert start == now - datetime.timedelta(seconds=10)
+    assert end == now
+
+    # Test when start and delta are provided
+    start, end = get_start_end(
+        start=now - datetime.timedelta(seconds=10),
+        end=None,
+        delta=datetime.timedelta(seconds=1),
+    )
+
+    assert start == now - datetime.timedelta(seconds=10)
+    assert end == start + datetime.timedelta(seconds=1)
+
+    # Test when start is bigger than end
+    with pytest.raises(
+        mlrun.errors.MLRunInvalidArgumentError,
+    ) as exc:
+        get_start_end(
+            start=now + datetime.timedelta(seconds=10),
+            end=now,
+        )
+        assert "The start time must be before the end time" in str(exc.value)
```

Comment:
The assert isn't actually getting run at all the way it's currently written. And if an exception were not raised, you'd get this error:
```
self = <ExceptionInfo for raises contextmanager>

    @property
    def value(self) -> E:
        """The exception value."""
        assert (
>           self._excinfo is not None
        ), ".value can only be used after the context manager exits"
E       AssertionError: .value can only be used after the context manager exits
```
The fix:
```suggestion
    with pytest.raises(
        mlrun.errors.MLRunInvalidArgumentError,
        match="The start time must be before the end time"
    ):
        get_start_end(
            start=now + datetime.timedelta(seconds=10),
            end=now,
        )
```